cmd/scheduler/main_test.go

```diff
@@ -418,224 +423,171 @@ profiles:
 		t.Fatal(err)
 	}
 
-	defaultPlugins := map[string][]kubeschedulerconfig.Plugin{
-		"QueueSortPlugin": {
-			{Name: "PrioritySort"},
-		},
-		"PreFilterPlugin": {
-			{Name: "NodeResourcesFit"},
-			{Name: "NodePorts"},
-			{Name: "PodTopologySpread"},
-			{Name: "InterPodAffinity"},
-			{Name: "VolumeBinding"},
-			{Name: "NodeAffinity"},
-		},
-		"FilterPlugin": {
-			{Name: "NodeUnschedulable"},
-			{Name: "NodeName"},
-			{Name: "TaintToleration"},
-			{Name: "NodeAffinity"},
-			{Name: "NodePorts"},
-			{Name: "NodeResourcesFit"},
-			{Name: "VolumeRestrictions"},
-			{Name: "EBSLimits"},
-			{Name: "GCEPDLimits"},
-			{Name: "NodeVolumeLimits"},
-			{Name: "AzureDiskLimits"},
-			{Name: "VolumeBinding"},
-			{Name: "VolumeZone"},
-			{Name: "PodTopologySpread"},
-			{Name: "InterPodAffinity"},
-		},
-		"PostFilterPlugin": {
-			{Name: "DefaultPreemption"},
-		},
-		"PreScorePlugin": {
-			{Name: "InterPodAffinity"},
-			{Name: "PodTopologySpread"},
-			{Name: "TaintToleration"},
-			{Name: "NodeAffinity"},
-		},
-		"ScorePlugin": {
-			{Name: "NodeResourcesBalancedAllocation", Weight: 1},
-			{Name: "ImageLocality", Weight: 1},
-			{Name: "InterPodAffinity", Weight: 1},
-			{Name: "NodeResourcesLeastAllocated", Weight: 1},
-			{Name: "NodeAffinity", Weight: 1},
-			{Name: "NodePreferAvoidPods", Weight: 10000},
-			{Name: "PodTopologySpread", Weight: 2},
-			{Name: "TaintToleration", Weight: 1},
-		},
-		"BindPlugin":    {{Name: "DefaultBinder"}},
-		"ReservePlugin": {{Name: "VolumeBinding"}},
-		"PreBindPlugin": {{Name: "VolumeBinding"}},
-	}
-
 	testcases := []struct {
 		name            string
 		flags           []string
 		registryOptions []app.Option
-		wantPlugins     map[string]map[string][]kubeschedulerconfig.Plugin
+		wantPlugins     map[string]*config.Plugins
 	}{
 		{
 			name: "default config",
 			flags: []string{
 				"--kubeconfig", configKubeconfig,
 			},
-			wantPlugins: map[string]map[string][]kubeschedulerconfig.Plugin{
-				"default-scheduler": defaultPlugins,
+			wantPlugins: map[string]*config.Plugins{
+				"default-scheduler": defaults.PluginsV1beta2,
```

Comment:
```suggestion
				"default-scheduler": defaults.PluginsV1beta1,
```

Reply:
This should stay v1beta2 as we vendor k8s 1.22 now. And in 1.22 a legacy config will be using v1beta2.

Reply:
I see. I wonder if we need to change other places to keep consistent with defaults.PluginsV1beta2. Like the following lines.

Reply:
For other tests, the out-of-tree plugin args is only available in v1beta1 yet. I will add v1beta2 args and then update the test.

Reply:
Thanks for your explanation